projects/github-actions/required-review/action.yml

```diff
@@ -1,5 +1,8 @@
-name: Required review
-description: Check that a PR has reviews from required teams.
+name: Required Review
+description: Check that a Pull Request has reviews from required teams.
+branding:
+  icon: 'git-pull-request'
```

Comment:
Personally I still like user-check for this one. git-pull-request seems fairly generic.